wcomponents-theme-parent/pom.xml

```diff
@@ -265,6 +268,27 @@
 							<goal>run</goal>
 						</goals>
 					</execution>
+					<!-- Build the i18n Resources from the wcomponents-i18n module. -->
```

Comment:
@ricksbrown This is where the "manual" build of I18N is done. This duplicates the build in wcomponents-i18n module. Thsi gets around the issue of the wcompoentns-i18n" jar not being packed before the test phase. This is not an issue for projects that consume wcomponents as wcomponents-i18n is already in the repo.